instances/atlas/brainAtlas/Desikan-KillianyAtlas.jsonld

```diff
@@ -0,0 +1,165 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/brainAtlas/AAL1",
```

Comment:
This is wrong, isn't it? Should be something like ".../brainAtlas/DKA", right?

Reply:
yes.... copy/paste error.... it is corrected now